Asset Uploader UI Component/Code/Asset Uploader/Objective-C/AssetUploader/ViewController.m

```diff
@@ -289,10 +343,12 @@ - (void)setupAdobeLibraryManager:(AdobeLibraryDownloadPolicyType)downloadPolicy
     rootLibDir = [rootLibDir stringByAppendingPathComponent:[NSBundle mainBundle].bundleIdentifier];
```

Comment:
Now that a user-specific UUID is generated for the Library directory, I don’t think this bundle identifier is needed any mote. I think it could be removed. The `Caches` directory will be app-specific and sandboxed, anyway, so this just makes the path long.